spec/unit/provider/service/launchd_spec.rb

```diff
@@ -345,4 +345,26 @@
       expect { provider.jobsearch("NOSUCH") }.to raise_error(Puppet::Error)
     end
   end
+
+  describe "read_overrides" do
+    before do
+      provider.stubs(:sleep)
```

Comment:
Think this should be `Kernel.stubs(:sleep)` now.

Reply:
Done.